01_introduction_to_algorithms/PowerShell/01_binary_search/Search-Binary.ps1

```diff
@@ -0,0 +1,53 @@
+﻿$mylist = (1,3,5,7,9);
+
+
+function Search-Binary
+{
+    #[CmdletBinding()]
+    #[Alias()]
+    #[OutputType([int])]
+    Param
+    (
+        #  numeric lists 
+        [Parameter(Mandatory=$true,
+                   ValueFromPipelineByPropertyName=$true,
+                   Position=0)]
+        
+        [int[]]$list,
+
+        # $item will then be passed between calls as you reduce your list
+        
+        $item
+    )
+
+    [int] $low = 0;
+    $high = $list.Length -1;
+
+    while ($low -lt $high) {
+
+        $mid = ($low + $high);
```

Comment:
This should be `$mid = ($low + $high) / 2;`. The code is wrong in the book (it is a known bug).